lisa/transformers/kernel_installer.py

```diff
@@ -205,20 +205,25 @@ def install(self) -> str:
         node: Node = self._node
         ubuntu: Ubuntu = cast(Ubuntu, node.os)
         release = node.os.information.codename
+        repo_component = "restricted main multiverse universe"
 
         assert (
             release
         ), f"cannot find codename from the os version: {node.os.information}"
 
         # add the repo
         if runbook.is_proposed:
-            version_name = f"{release}-proposed"
+            if ( "proposed2" in self.repo_url or "private-ppa" in self.repo_url):
+                version_name = release
+                repo_component = "main"
```

Comment:
What's the reason removes other parts, like restricted multiverse universe? Please add comment in code.